lib/urbanopt/geojson/schema/building_properties.json

```diff
@@ -520,11 +520,11 @@
         "DEER 2065",
         "DEER 2070",
         "DEER 2075",
-        "IECC 2006",
-        "IECC 2009",
-        "IECC 2012",
-        "IECC 2015",
-        "IECC 2018"
+        "Residential - IECC 2006 / EnergyStar Sep 2020",
```

Comment:
Change to "Residential 2006", "Residential 2009", ...? Something we could leave as a final task. Or "Residential 2009 Customizable Template"- users can go in and modify the `mappers/residential/*.tsv` files.